.gen/pipeline/api/openapi.yaml

```diff
@@ -11813,6 +11813,139 @@ paths:
       summary: Sync ARK backups of an Organization
       tags:
       - ark-backups
+  /api/v1/orgs/{orgId}/clusters/{id}/update:
+    put:
+      description: Update a cluster
+      operationId: ClusterUpdate
+      parameters:
+      - description: Organization identifier
+        explode: false
+        in: path
+        name: orgId
+        required: true
+        schema:
+          type: integer
+        style: simple
+      - description: Cluster identifier
+        explode: false
+        in: path
+        name: id
+        required: true
+        schema:
+          type: integer
+        style: simple
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ClusterUpdateRequest'
+        required: true
+      responses:
+        "202":
+          description: Accepted
+        default:
+          content:
+            application/json:
+              examples:
+                BadRequest:
+                  value:
+                    code: 400
+                    message: Invalid request
+                    error: Failed to parse request.
+                Unauthorized:
+                  value:
+                    code: 401
+                    message: Unauthorized
+                    error: Unauthorized
+                OrganizationNotFound:
+                  value:
+                    code: 404
+                    message: Organization not found
+                    error: Organization not found
+                SecretAlreadyExists:
+                  value:
+                    code: 409
+                    message: Secret already exists
+                    error: Secret already exists
```

Comment:
**Note**: is this the correct message for a 409 Conflict here?
I know it's auto-generated and not manually specified, but it might be worth an investigation when we have the time, although it is out of the scope of this PR I guess.

Same at
https://github.com/banzaicloud/pipeline/pull/3123/files#diff-2b6bfc5b5d040867ca5e5f7b1df191f1R11921

Reply:
This is the generated descriptor. Please comment on the original one.

Reply:
Follow up at https://github.com/banzaicloud/pipeline/pull/3123/files#r478930068